src/sass/prefooter.scss

```diff
@@ -1,26 +1,30 @@
+nav#sub-navigation {
```

Comment:
Can you put this inside of the namespace?

Reply:
Done.